settings.gradle

```diff
@@ -61,7 +61,8 @@ include ":opentelemetry-examples-autoconfigure",
         ":opentelemetry-examples-sdk-usage",
         ":opentelemetry-examples-telemetry-testing",
         ":opentelemetry-examples-zipkin",
-        ":opentelemetry-examples-spring-native"
+        ":opentelemetry-examples-spring-native",
+        ":kotlin-extension"
```

Comment:
all other modules start with `:opentelemetry-examples-` whiles yours does not, consider following established conventions.

Reply:
Thank you, I've updated it to match the other module names.